pkg/controller/schedulingmanager/controller.go

```diff
@@ -47,10 +47,10 @@ type SchedulerController struct {
 
 	scheduler map[string]schedulingtypes.Scheduler
 
-	config   *util.ControllerConfig
-	stopChan <-chan struct{}
+	config *util.ControllerConfig
 
 	runningPlugins sets.String
+	templateKinds  map[string]string
```

Comment:
This map appears to associate a template kind with a scheduling kind.  I don't think this is obvious from the name `templateKinds` and encourage you to either choose a more descriptive name or add a comment to document your intent.

(No action required) As per a subsequent comment, I'm not sure why the scheduling preference controller shouldn't be tracking the association between scheduling and template kinds.